src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -252,18 +252,6 @@ private static OtlpTrace.Status ToOtlpStatus(ref TagEnumerationState otlpTags)
                 Code = (OtlpTrace.Status.Types.StatusCode)(int)status,
             };
 
-            if (otlpStatus.Code != OtlpTrace.Status.Types.StatusCode.Error)
-            {
-#pragma warning disable CS0612 // Type or member is obsolete
-                otlpStatus.DeprecatedCode = OtlpTrace.Status.Types.DeprecatedStatusCode.Ok;
```

Comment:
We could update the protos - this field was removed in 0.12.0. Our protos are 0.11.0. Though, I’d be happy approving this PR and I could follow up with that if you’d prefer.

Reply:
let me do it as next PR.